panel/io/jupyter_server_extension.py

```diff
@@ -265,13 +265,11 @@ async def get(self, path=None):
 
         kernel_env = {**os.environ}
         kernel_id = await ensure_async(
-            (
```

Comment:
Maybe reindent this

Reply:
Done in e5c42e9116a1311341ff4a78636efbaf9f0b67f8